hypha/apply/projects/templates/application_projects/includes/supporting_documents.html

```diff
@@ -53,7 +53,9 @@ <h5 class="docs-block__heading">{% trans "Supporting documents" %}</h5>
                 {% if editable %}
                     <a class="docs-block__link" href="{% url 'apply:projects:edit' pk=object.pk %}">
                         {% if object.user_has_updated_details %}
-                        {% trans "Edit" %}
+                            {% if not user.is_applicant %}
+                                {% trans "Edit" %}
+                            {% endif %}
```

Comment:
Quoting from the issue description..

>It can stay in the list to show the checkmark but applicant should not have any access to view or edit it.

@frjo just to confirm...are the applicant allow to fill the approval form?

Btw it would be nice check what permissions user should have to do this instead of just relying on user not being the applicant.

@frjo what permission do we need to check for the user who can submit approval form/or what do we call those user? Approver? 

Will the checks happening for [user_can_approve_project](https://github.com/HyphaApp/hypha/blob/e97d7ea1d494a89cd4285961a166e34e2c3a9293/hypha/apply/projects/templatetags/approval_tools.py?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L18) be enough/correct for this?

Reply:
It is only staff that are allowed to fill in the project approval form (PAF). And to approve the form a user must be staff + approver role. Staff and Finance are both allowed to view the project approval.

Applicants should not have any access, view/edit/delete or otherwise, to the project approval form.

Reply:
@frjo I have changed the condition. Can you please check it again, It should work fine now.